discovery-infra/test_infra/helper_classes/kube_helpers.py

```diff
@@ -0,0 +1,376 @@
+import logging
+import os
+import contextlib
+
+import waiting
+
+from typing import Optional, Union, Dict, Tuple, ContextManager
+
+from kubernetes.config import load_kube_config
+from kubernetes.config.kube_config import Configuration
+from kubernetes.client import ApiClient, CoreV1Api, CustomObjectsApi
+from kubernetes.client.rest import ApiException
+
+from tests.conftest import env_variables
+from test_infra.utils import get_random_name
+
+
+# silence kubernetes debug messages.
+logging.getLogger('kubernetes').setLevel(logging.INFO)
+
+logger = logging.getLogger(__name__)
+
+
+def _does_string_contain_value(s: Union[str, None]) -> bool:
+    return s and s != '""'
+
+
+class ObjectReference(dict):
+
+    def __init__(self, name: str, namespace: str):
+        dict.__init__(self, name=name, namespace=namespace)
+
+
+class _BaseKubeAPIResource(object):
```

Comment:
You don't have to explicitly inherit from `object` since all the classes inherit from `object` by default for Python3

Reply:
sure, just like to be explicit, is it critical?

Reply:
Not critical. Just unnecessary.

Reply:
removed